README.md

```diff
@@ -13,6 +13,7 @@ and applications on [Google App Engine](http://cloud.google.com/nodejs).
 ### Frameworks
 
 - Express.js - [Source code][express_1] | [App Engine Tutorial][express_2] | [Live demo][express_3] | [Documentation][express_4]
+  - Express.js + Memcached Sessions - [Source code][express_5] | [Documentation][express_6]
```

Comment:
nit that you don't have to fix now: it would be better to have more described anchors here. `express_6` is kinda of funny.